ui/templates/robots.txt

```diff
@@ -0,0 +1,15 @@
+User-agent: *
+Disallow: /dashboard/
+Disallow: /learner/
+Disallow: /learners/
+Disallow: /cms/
+Disallow: /profile/
+Disallow: /financial_aid/
+Disallow: /admin/
+Disallow: /logout/
+Disallow: /404/
+Disallow: /500/
+Disallow: /terms_of_service/
```

Comment:
No. Let's keep all of those. Robots need to get errors, and they know what to do with them.